ui/redux/actions/comments.js

```diff
@@ -54,58 +57,97 @@ export function doCommentList(
     const creatorChannelClaim = getChannelFromClaim(claim);
     const { claim_id: creatorClaimId, name: channelName } = creatorChannelClaim || {};
 
-    return Comments.comment_list({
-      page,
-      claim_id: claimId,
-      page_size: pageSize,
-      parent_id: parentId,
-      top_level: !parentId,
-      channel_id: creatorClaimId,
-      channel_name: channelName,
-      sort_by: sortBy,
-    })
-      .then((result: CommentListResponse) => {
-        const { items: comments, total_items, total_filtered_items, total_pages } = result;
+    let channelSignature = {};
+    if (isProtected) {
```

Comment:
:small_red_triangle: The entire `comment_list` call is moved under `isProtected` -- wouldn't that break comment-fetching for the normal case (`isProtected === false`)?

<img width="557" alt="image" src="https://user-images.githubusercontent.com/64950861/191402613-8e92491b-78f1-4aad-8dab-42f3152ce829.png">